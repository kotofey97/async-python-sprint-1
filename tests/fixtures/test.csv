Город/день,,2022-05-26,2022-05-27,2022-05-28,2022-05-29,2022-05-30,Среднее
MOSCOW,"Температура, среднее",18.0,13.1,12.2,12.0,---,13.82
,"Без осадков, часов",4.0,0.0,0.0,1.0,---,1.25
CAIRO,"Температура, среднее",32.6,33.2,34.5,---,---,33.43
,"Без осадков, часов",10.0,10.0,10.0,---,---,10.0
NOVOSIBIRSK,"Температура, среднее",24.9,22.3,23.1,22.2,---,23.13
,"Без осадков, часов",10.0,10.0,6.0,2.0,---,7.0
BUCHAREST,"Температура, среднее",27.5,26.0,27.9,18.0,---,24.85
,"Без осадков, часов",10.0,10.0,9.0,0.0,---,7.25
