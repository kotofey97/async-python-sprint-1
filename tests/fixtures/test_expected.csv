Город/день,,2022-05-26,2022-05-27,2022-05-28,2022-05-29,2022-05-30,Среднее
MOSCOW,"Температура, среднее",18.0,13.1,12.2,12.0,---,13.82
,"Без осадков, часов",4,0,0,1,---,1.25
CAIRO,"Температура, среднее",32.6,33.2,34.5,---,---,25.08
,"Без осадков, часов",10,10,10,0,---,7.5
NOVOSIBIRSK,"Температура, среднее",24.9,22.3,23.1,22.2,---,23.13
,"Без осадков, часов",10,10,6,2,---,7.0
BUCHAREST,"Температура, среднее",27.5,26.0,27.9,18.0,---,24.85
,"Без осадков, часов",10,10,9,0,---,7.25
